console-nginx/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM nginxinc/nginx-unprivileged:1.19.6-alpine
```

Comment:
What's the point of having that image re-builded here? Why not just replicating the image directly with the image-syncer

Reply:
It's just a base to start work with. I'm going to modify and extend the image.

Reply:
So this PR in it's current shape brings no value value, please do not merge it till there is a substantle improvement visible ;)

btw, if you re-build the image anyway, then you should think of making it based on alpine in it's latest version and copy over the binaries. That will assure that latest packages are getting used with less security vulnerabilities in the base image. And you have the chance at any time to update the base image version independent of the upstream nginx release cycle. That might be a first valuable step to start with.

Reply:
Changed